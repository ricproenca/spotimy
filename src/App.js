import React from 'react';
import { hot } from 'react-hot-loader/root';

/**
 * App
 */
const App = () => {
  return <h1>SPOTIMY</h1>;
};

export default hot(App);
