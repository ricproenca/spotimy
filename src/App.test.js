import { cleanup, render } from '@testing-library/react';
import React from 'react';

import App from './App';

describe('App', () => {
  afterEach(cleanup);

  it('should render without crashing', () => {
    // Arrange

    // Act
    const { asFragment } = render(<App />);

    //Assets
    expect(asFragment()).toMatchSnapshot();
  });
});
