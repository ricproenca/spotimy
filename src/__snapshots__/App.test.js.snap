// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<DocumentFragment>
  <h1>
    SPOTIMY
  </h1>
</DocumentFragment>
`;
